src/pretalx/event/models/event.py

```diff
@@ -781,16 +781,17 @@ def get_date_range_display(self) -> str:
         """
         return daterange(self.date_from, self.date_to)
 
-    def release_schedule(self, name: str, user=None, notify_speakers: bool = False):
+    def release_schedule(self, name: str, comment: str, user=None, notify_speakers: bool = False):
```

Comment:
Please make the comment optional, defaulting to ``None``, and move it to the end of the method signature – that way less tests should break, too.